Documentation/docs/contributing/python_packaging.md

```diff
@@ -0,0 +1,148 @@
+Build ITK Python packages
+======================================
+
+We can generate Python wrappings and package them as Python wheels in order to conveniently distribute and use the ITK C++ library in Python scripts. In most cases community members can use the [general ITK Python distributions](https://pypi.org/project/itk/) on the Python Package Index (PyPI) for their projects.
+
+```py
+> pip install itk
+```
+
+In some cases developers may need to build ITK Python wrappings for local testing without distribution. Visit the ITK Software Guide "Wrapping" section to learn more about that process.
+
+Additionally, InsightSoftwareConsortium maintainers build ITK Python packages with each release candidate on Kitware build systems and upload them to
+the [ITKPythonPackage GitHub releases page](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/releases).
+
+This section describes how to build Python packages for the main ITK libraries using the [ITKPythonPackage](https://github.com/insightSoftwareConsortium/ITKpythonpackage) project. Follow the steps below to reproduce the build process used to generate the official ITK Python release packages.
+
+## Prerequisites
+
+Building ITK Python wheels requires the following:
+- CMake >= 3.16
+- Git
+- C++ Compiler (see [scikit-build platform specific requirements](https://scikit-build.readthedocs.io/en/latest/generators.html))
+- Python >= 3.8
+
+## Automated Platform Scripts
+
+The following sections outline how to use the ITKPythonPackage project to build wheels on Linux, macOS, and Windows. Each script will fetch tagged ITK sources, build ITK with Python wrappings, and package binaries into the Python wheel archive format for distribution.
+
+### Linux
+
+On any linux distribution with docker and bash installed, running the script [dockcross-manylinux-build-wheels.sh](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/dockcross-manylinux-build-wheels.sh) will create 64-bit wheels for Python 3.x in the `dist` directory. A Python version to target may be passed as a trailing argument, or passing no arguments will target all default Python versions.
+
+In addition, the environment variables `MANYLINUX_VERSION` and `IMAGE_TAG` may be set before calling the script to control the [dockcross](https://github.com/dockcross/dockcross) Docker image and target a specific platform type. See ITKPythonPackage for more information.
+
+For example:
+
+```sh
+$ git clone https://github.com/InsightSoftwareConsortium/ITKPythonPackage.git
+[...]
+
+$ pushd ITKPythonPackage
+$ export MANYLINUX_VERSION=2014
+$ ./scripts/dockcross-manylinux-build-wheels.sh cp38
+[...]
+
+$ ls -1 dist/
+itk-5.3.0.dev20231108-cp38-cp38m-manylinux2014_x86_64.whl
```

Comment:
Can we remove the `$`?

Reply:
It is useful to understand the build command output here. Do you have an alternate suggestion on how to convey input/output?

Reply:
Yes, we could keep these.